### ISSUE TITLE
Objection to changing IRIs for previously defined terms 

### ISSUE THREAD

**Issue by alanruttenberg** [OBOFoundry/COB#81]
I see that COB currently has new IRIs for many already defined terms. This is bad practice and should not be done.

**Comment by hoganwr** [comment]
I agree. We just need a technical solution for resolving them properly, and James has a working prototype.

**Comment by jamesaoverton** [comment]
Since the COB 2024 workshop in particular, we are always either reusing existing terms or minting new terms.
